share/picongpu/tests/FieldAbsorber/bin/ci.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#
+# This file is part of PIConGPU.
+# Copyright 2023 PIConGPU contributors
+# Authors: Mika Soren Voss, Hannes Wolf, Klaus Steiniger
+# License: GPLv3+
+#
+
+set -o pipefail
```

Comment:
As far as I know, this line makes the script stop whenever an instruction fails. This is not limited to pipes. And I think I added it later during development, because I noted some early errors only later during execution.

Reply:
(for example, when `cd`ing in a directory that does not exist. The script continues running when pipefail is not set)

Reply:
No, this only applies to pipes - would be nice to have something like that for fails as well, but then catching non-zero returns would not work anymore. 
Not that important since one might use pipes in the future. This line does not hurt.